website/docs/advanced-concepts/multiple-environments.md

```diff
@@ -0,0 +1,95 @@
+
+# Multiple Environments
+
+Environments are container entities within a workspace that let you create different updates to different versions of your content types while still keeping them all in isolation. This process is often utilized when developing, identifying errors, or introducing new features without any concerns about how it might influence the current production.
```

Comment:
This intro looks a little out of context. Can we keep the one shared by the engineering team in the first draft for now.

>  While building apps, there is usually a clear distinction between the version of the app that end users see versus the version that is still under development or testing. Most pieces of software of reasonable complexity make this distinction clear by having different “environments” for development, testing and production use. This ensures that end users of the app are not impacted by changes which are still in development.

Reply:
updated